README.md

```diff
@@ -45,8 +45,31 @@ For other systems, please check the above requirements and use your correspondin
 TBD
 
 ## Windows
-
-TBD
+### GL-Node Builder
+- get [glbsp executable](http://glbsp.sourceforge.net/download.php)
+- place it in the root of the goom directory
+
+### mingw-w64 for makefile execution and portmidi usage
```

Comment:
This is doc, avoid long technical title lines.
Also Makefiles always start with a capital M ;)